core/templates/domain/admin/beam-job-run.model.ts

```diff
@@ -0,0 +1,180 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
I'm not sure about the structure, but I think it would be better to have one bean-job.model.ts  (or job.model) file with all the jobs related classes in one file.

Reply:
@srijanreddy98 Do you have any suggestion?

Reply:
Also, not sure whether we should keep theses models in /admin/ dir*

Reply:
I am actually fine with any. If having them in one file is a requirement, @brianrodri you can put them all in a folder (lets say jobs) and have a barrel file to export all these from one file. So imports from outside would be from domain/admin/jobs.

Reply:
I'm fine with either approach, I lean slightly towards separate models-per-file because they're all conceptually different.

Reply:
In that case, let's have a domain/job/ dir and have a separate file for each model. @brianrodri is that fine?

Reply:
Done.